distributed/cli/dask_scheduler.py

```diff
@@ -125,6 +126,10 @@ def del_pid_file():
                           scheduler_file=scheduler_file,
                           security=sec)
     scheduler.start(addr)
+    if not preload:
+        preload = dask.config.get('distributed.scheduler.preloads')
+    if not preload_argv:
+        preload_argv = dask.config.get('distributed.scheduler.preloads-argv')
```

Comment:
Thoughts on preload vs preloads?  I'm not sure if we should pluralize this word, given the single-ness of the `--preload` CLI keyword.

Reply:
Fine by me to go with preload, I'll update the PR accordingly.